cmd/api/src/api/middleware/middleware_internal_test.go

```diff
@@ -80,35 +80,51 @@ func TestRequestWaitDuration(t *testing.T) {
 	require.True(t, requestedWaitDuration.UserSet)
 }
 
-func TestParseUserIP_XForwardedFor(t *testing.T) {
+func TestParseUserIP_XForwardedForMissing(t *testing.T) {
 	req, err := http.NewRequest("GET", "/teapot", nil)
 	require.Nil(t, err)
 
-	ip1 := "192.168.1.1:8080"
-	ip2 := "192.168.1.2"
-	ip3 := "192.168.1.3"
-	req.Header.Set("X-Forwarded-For", strings.Join([]string{ip1, ip2, ip3}, ","))
+	req.RemoteAddr = "http://www.google.com/0.0.0.0:3000"
```

Comment:
The `http.Request.RemoteAddr` field says it's just `<host>:<port>` so I believe this should just be `'0.0.0.0:3000'`